workshop/wordpress/porter.yaml

```diff
@@ -1,6 +1,6 @@
-name: wordpress
+name: porter-workshop-wordpress
```

Comment:
Sam here maybe just call it `workshop-wordpress`. I think the only special bundle that gets to have the porter- prefix is `porter-hello`.